src/main/java/org/generationcp/middleware/dao/GermplasmDAO.java

```diff
@@ -106,6 +105,25 @@
 
 	private static final String FIND_GERMPLASM_BY_GIDS = " g.gid in (:gids) ";
 
+	private static final String FIND_GERMPLASM_WITHDESCENDANTS = "SELECT g.gid as gid FROM germplsm g "
```

Comment:
Is this still used?

Reply:
Yes, Used by getGidsOfGermplasmWithDescendants